### PR TITLE
[iTunes] Store country or region specification

### PR DESCRIPTION
**Pull request by wolther47**
Add an optional country or region code in the URL path, see #1503 for details.

### CODE REVIEW

**Comment by RedSparr0w** [comment]
Thanks for the PR,

It looks like the tests aren't running as expected though.
Not sure why it does this, but do you happen to have shields setup on your own Circle CI?
If so, could you remove it from your own and that should let the tests run as expected.

File: server.js [hunk 1]
```diff
@@ -2048,12 +2048,18 @@ cache(function(data, match, sendBadge, request) {
...
+camp.route(/^\/itunes\/v\/(?:(.+)\/)?(.+)\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
-  var bundleId = match[1];  // eg, `324684580`
-  var format = match[2];
-  var apiUrl = 'https://itunes.apple.com/lookup?id=' + bundleId;
+  var region = match[1]; // eg, `us`
+  var bundleId = match[2];  // eg, `324684580`
+  var format = match[3];
+
+  var apiUrl = (region == '')
```
Comment (chris48s) [inline server.js]: Can you edit this to use a strict comparison here (triple equals).

Also I know they were `var` when you started, but while you're editing this block of code, could you update these variable declarations to use the ES6 style `const` please.

**Comment by chris48s** [comment]
I've added a minor comment to have a look at. In addition to @RedSparr0w 's comment on fixing the build, could you also rename the pull request to "[itunes] my pull request title" so that CircleCI knows to run the tests for the itunes service next time the build runs (see https://github.com/badges/shields/blob/master/service-tests/README.md#pull-requests ).
Thanks

**Comment by paulmelnikow** [comment]
Are we going to address this as part of #1515? If so, let's close this to make sure we're not duplicating any work!

**Comment by paulmelnikow** [comment]
Closing for inactivity. If you're reading this and want this implemented, don't be discouraged. You can grab the commits from the pull request branch using `git fetch origin pull/1515/head:pr-1515`, address the comments, and reopen.

**Pull request closed without merging**